main.c

```diff
@@ -302,12 +307,20 @@ int main(void) {
 
 	imu_reset_orientation();
 
-#ifdef BOOT_OK_GPIO
 	chThdSleepMilliseconds(500);
```

Comment:
I did notice this as well, but left it like this for now.

If the delay is needed it should always be there and not only when the boot pin is active. It does not affect the startup time for most applications, only the time until the blocking terminal thread and the lisp code starts in order to make sure that all things are done with their initialization routines. Most likely it can be reduced or removed.

Looking into the boot time in general, especially the offset measurement, is on my todo list, I will get back to it soon.